probe-rs/src/architecture/arm/core/armv7a.rs

```diff
@@ -963,6 +980,10 @@ mod test {
             Ok(())
         }
 
+        fn read(&mut self, _ap: MemoryAp, _address: u64, _data: &mut [u8]) -> Result<(), Error> {
+            todo!()
```

Comment:
I think you are confusing things. I definitely put this `todo!` only as a placeholder and it wasn't different before. I will change it :)

Reply:
Hmm, this is just the `MockProbe`. I think a `todo!()` should be fine and might even be preferrable since we can implement it properly if we ever need it and it's obvious when tests fail.